setup.py

```diff
@@ -132,7 +132,7 @@ def extra(self):
         'typing-extensions>=3.7.4.3'
     ],
     extras_require={
-        'email': ['email-validator>=1.0.3'],
+        'email': ['email-validator==1.1.3'],
```

Comment:
```suggestion
        'email': ['email-validator>=1.0.3,<1.2.0'],
```

is technically more correct.

Reply:
I strongly agree, I missed the syntax, thanks a lot 🙏

Reply:
just check the exact syntax, I might be wrong, but it's something like that.